app/views/scans/_form_additional.html.erb

```diff
@@ -1,5 +1,11 @@
-<%= f.text_field :page_range, help: 'Example: 1-5, 8, 7-13' %>
+<div class="form-group">
+  <%= f.label :page_range, class: 'control-label col-sm-3' %>
```

Comment:
We have a helper that has the label and content column sizes so we can change them all in one place if need be: https://github.com/sul-dlss/sul-requests/blob/master/app/helpers/application_helper.rb#L19-L21